src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/X509Pal.PublicKey.cs

```diff
@@ -32,6 +32,11 @@ public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte
                 return DecodeECDsaPublicKey((CertificatePal)certificatePal);
             }
 
+            return DecodePublicKey(oid, encodedKeyValue, encodedParameters);
+        }
+
+        public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte[] encodedParameters)
```

Comment:
I really want only one public overload.  If/when we can get rid of the pal hook for the public key for ECC we'd remove that parameter from the signature.

Specifically, my concern is that someplace which should support ECC now might not, because of not realizing the difference in the two exposed methods.